src/main/java/app/controllers/ProfessorController.java

```diff
@@ -105,16 +118,95 @@ public ModelAndView saveProject(@AuthenticationPrincipal UserDetails currentUser
         return new ModelAndView("redirect:/facultyMenu");
     }
 
-    // Not currently working. Need to change the confirmation workflow
+
+    /**
+     * Navigate to the Add Student Page
+     * @param model Model
+     * @param projectID Project ID to link Student
+     * @return Redirect to the "new-student" page
+     */
+    @RequestMapping("/new-student/{projectID}")
```

Comment:
the route should follow the convention project/{id}/new-student
this is because the id is related to the projects and the new-student is an action on project with {id}

Reply:
Done